e2e-tests/steps/base-steps.js

```diff
@@ -1,6 +1,14 @@
 export class BaseSteps {
-  static visit(page) {
-    cy.visit(page);
+  static visit(page, stubNewWindow = false) {
+    if (stubNewWindow) {
+      cy.visit(page, {
+        onBeforeLoad(win) {
+          cy.stub(win, 'open').as('window.open');
```

Comment:
I don't like having the stub in the steps.

It must be stubbed after the visit but I don't think it should be here.

I have in my MR stubbed it also. Check how it is there to discuss it.
https://github.com/Ontotext-AD/graphdb-workbench/pull/2164/files#diff-55d7fb77568da8df62a4007e9d5c0f4eedad6b25122921ffbb2160271d843bc7